### DISCUSSION TITLE
Makdown cells

### DISCUSSION THREAD

**Discussion by ratepenado** [fonsp/Pluto.jl#1635] [Questions about using Pluto]
Hello,
Is there any plan to have specific markdown cells like one can find in jupyterlab ? 

Thanks !

**Comment by disberd** [comment]
What would be the advantage to a cell wrapped in md"""""""?

**Reply by ratepenado** [reply]
A better interface to write markdown. You can see the difference with jupyter in these two screenshots: https://aws1.discourse-cdn.com/business5/uploads/julialang/original/3X/1/0/10943cf951784bc4790225a47a47f9256a3c0b56.png and https://www.earthdatascience.org/images/courses/earth-analytics/bootcamp/jupyter-interface/change-cell-type.png

**Reply by disberd** [reply]
Unfortunately both of your image links lead me to a github 404 not found :(

Edit: I notice now that the hyperlinks associated to the urls text are wrong and not matching the URL names you linked, copying instead of clicking works.

This pull request https://github.com/fonsp/Pluto.jl/pull/1533 seems to provide the automatic markdown synthax highlighting within md"" blocks as you seem to have in the linked picture

**Reply by ratepenado** [reply]
sorry ! I probably inverted url and url names.

**Comment by dralletje** [comment]
We've successfully avoided cell types so far 🙃 

First we want to explore how far we can get with having the formatting like you shown in the jupyter screenshot, but inside the `md"""..."""` blocks. This is harder to do right, but if we do it right it will be very cool. 

Sure, cell types are also a possibility. Actually Observablehq has a pretty nice UI for that at the moment.
